### ISSUE TITLE
Issue tocreate attachment-logs with special filenames

### ISSUE THREAD

**Issue by crackytsi** [TheHive-Project/TheHive#1126]
### Request Type
Bug 

### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | RHEL
| OS version (client)       | Seven
| TheHive version / git hash   | 3.4

### Problem Description
I try to attache a file with the name "复答 test.pdf" using Python requests.
Unforunately this is not possible.

I'm not absolutly sure if this a failure on TheHive or on python-requests.
Can you help me?
"usual" characters works as expected, by doing some tcpdumps I also did not get on the right track.

The following issue brought me to think about if there is a possible encoding issue with RFC 2231 encoding of "special characters".
Can you take a look on this?
https://github.com/psf/requests/issues/3046

```
"/api/case/task/<taskid>/log"
f = {'attachment': ( os.path.basename(filename), open(file, 'rb'), magic.Magic(mime=True).from_file(file))}
header = {'Authorization': self.authkey}
requests.post(req, data=data,files=f, headers=header)

```

**Comment by github-pba** [comment]
True. Currently, there is an issue with Unicode characters in file names. Already reported as 1068.
